workflow/rules/annotations.smk

```diff
@@ -1,40 +1,36 @@
 rule annotations:
     input:
         expand("resources/mitos_db/{refseq}", refseq=mitos_refseq),
-        "results/assembled_sequence/{sample}.ok",
+        fasta="results/assembled_sequence/{sample}.fasta",
     params:
         refseq=mitos_refseq,
         code=mitos_code,
     output:
+        directory("results/annotations/{sample}/"),
         ok="results/annotations/{sample}/{sample}.ok",
```

Comment:
Hi, nice work, the code seems to work when tested on my computer,
I noticed you still have this last ".ok file" in the rules. by testing the code I understood that
this rule may not always output the `result.bed` file (or even does not create the output directory), but that you already have the checkpoint `extract_annotated_genes` to manage that. To remove the need for the .ok file you can extract (and make a new method similar to `get_annotated_samples` but using the checkpoint `extract_annotated_genes`) the list of successfully annotated samples (i.e. having a `result.bed` file). I can provide an example of working code in a fork, later this weekend if you wish.

Reply:
Hi @pecholleyn, well spotted, I left this.ok file because mitos can create a nested file directory if there is more than one assembled contig. I will see if I can get a working example.

Reply:
I committed a working piece of code [here](https://github.com/pecholleyn/skim2mt/tree/checkpoint_fix). Maybe it needs additional testing (e.g. without go_fetch)

Reply:
@o-william-white you can add the fork as a "remote" in your local repo (see https://www.atlassian.com/git/tutorials/syncing, basically a bookmark to another fork) and then checkout branches from it - e.g. `git checkout <remote>/<branch>`

Reply:
Hi @elichad, thanks for suggesting this, think I have a local copy of the branch now using the commands below. I will test the changes now. 

```
# add a remote git 
git remote add pecholleyn_checkpoint https://github.com/pecholleyn/skim2mt.git
git fetch pecholleyn_checkpoint
git checkout pecholleyn_checkpoint/checkpoint_fix
```